tests/unit/algorithm/RobustLineIntersectionTest.cpp

```diff
@@ -452,7 +445,7 @@ void object::test<10>
         "LINESTRING (-58.00593335955 -1.43739086465, -513.86101637525 -457.29247388035)",
         "LINESTRING (-215.22279674875 -158.65425425385, -218.1208801283 -160.68343590235)",
         1,
-        "POINT ( -215.22279674875 -158.65425425385 )",
+        "POINT ( -215.22279674875003 -158.65425425385004 )",
```

Comment:
Feels like a case for rounding? I guess not until we have some other platform that's got a slightly different answer...?

Reply:
Agreed, this would be more robust if the test specified a precision for the result.  More so in GEOS than JTS, since  there is more chance of platform-related discrepancy.

Reply:
Fixed by adding a distance tolerance.